scripts/manage-event-lifecycle.js

```
@@ -0,0 +1,452 @@
+  }
+  
+  try {
+    switch (action) {
+      case 'list':
+        listEvents();
+        break;
+        
+      case 'transition':
+        const newState = args[2];
+        if (!eventId || !newState) {
+          console.error('❌ Usage: transition <event-id> <state>');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)